hibernate-reactive-core/src/test/java/org/hibernate/reactive/ReactiveMultitenantNoResolverTest.java

```diff
@@ -227,6 +292,22 @@ public void testWithTransactionThrowsExceptionWithoutTenantWithMutiny(TestContex
 		test( context, getMutinySessionFactory().withTransaction( (s, t) -> selectCurrentDB( s ) ) );
 	}
 
+	@Test
+	public void testWithStatelessSessionThrowsExceptionWithoutTenant(TestContext context) {
+		thrown.expectCause( isA( HibernateException.class ) );
+		thrown.expectMessage( "no tenant identifier" );
+
+		test( context, getSessionFactory().withStatelessSession( this::selectCurrentDB ) );
+	}
+
+	@Test
+	public void testWithStatelessSessionThrowsExceptionWithoutTenantWithMutiny(TestContext context) {
+		thrown.expectCause( isA( CompletionException.class ) );
```

Comment:
This assertion is wrong for a couple of different reasons:
1. You are using the `Mutiny.SessionFactory` so there is not going to be a `CompletionException` anywhere (hopefully).
2. If you were using the `Stage.SessionFactory`, the error iwould be a `CompletionStageException` but the cause is going to be a `HibernateException` (like in `testWithStatelessSessionThrowsExceptionWithoutTenant`). Check the difference bewtween `expect` and `expectCause`.

You need to change this to:
```
	@Test
	public void testWithStatelessSessionThrowsExceptionWithoutTenantWithMutiny(TestContext context) {
		thrown.expect( isA( HibernateException.class ) );
		thrown.expectMessage( "no tenant identifier" );

		test( context, getMutinySessionFactory().withStatelessSession( this::selectCurrentDB ) );
	}
```
```